### PR TITLE
feat(misc, TextContent): update core version & text content updates

### LINKED ISSUE #10314: Updates to update and build latest core
We can't build the latest core currently. Steps to get us there:

[details to come]
```[tasklist]
### Tasks
- [x] Remove dropdown styles from menu
- [ ] https://github.com/patternfly/patternfly-react/issues/10316
- [ ] https://github.com/patternfly/patternfly-react/issues/10335
```

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #10314, #10262, #10335

- Updated core version to `alpha.135`
- Updated generateTokens, removing old, unused files it was pulling in to map vars
- Updated Text component suite to use default content classes (#10262)
- Updated Wizard to not use danger token directly & structure update (#10335)
- Added blurb to docs about Text changes
- Removed dropdown styles usage from Menu
- Updated Charts font weight token `global_FontWeight_bold` to `global_font_weight_heading_bold`
- Updated Slider inset token `c_slider__step_Left` to `c_slider__step_InsetInlineStart`
- Updated Td/Th inset tokens `c_table__sticky_cell_Left` and `c_table__sticky_cell_Right` to `c_table__sticky_cell_InsetInlineStart` and `c_table__sticky_cell_InsetInlineEnd`
- Updated Toolbar chip group class (variant prop still uses `chip-group`, but maps to `pf-m-label-group` now)
- Updated Wizard Footer to use ActionList (also for deprecated)
- Updated global height breakpoint token names
- Updated snapshots
- Fixed integration tests
- Updated Table integration demo token `global_primary_color_100` to `global_color_brand_default`
- Fixed Table transformers test referencing dropdown styles to menu styles
- Fixed react-token test, updated old token to new token

### CODE REVIEW

**Comment by kmcfaul** [comment]
I'm not sure if it's expected that a bunch of charts colors changed in the snapshots @mcoker Thoughts?

**Review by mcoker — COMMENTED** [review]
This looks great! A comment on the wizard footer action list. Looks like the structure between `<WizardFooter>` and `<WizardFooterInternal>` are different, and unless my eyes are playing tricks on me, I think they're both different from core. I'd expect that to be:

```
wizard-footer
  action-list
    action-list-group
      action-list-item // back
      action-list-item // next
    action-list-group
      action-list-item // cancel
```

I think that should future-proof from needing HTML changes if we add more destructive and/or non-destructive actions in the future. I can elaborate more on that if you'd like.

File: packages/react-core/src/components/Wizard/WizardToggle.tsx [hunk 1]
```diff
@@ -90,8 +94,14 @@ export const WizardToggle = ({
         aria-expanded={isNavExpanded}
       >
         <span className={css(styles.wizardToggleList)}>
-          <span className={css(styles.wizardToggleListItem)}>
-            <span className={css(styles.wizardToggleNum)}>{wizardToggleIndex}</span>{' '}
+          <span className={css(styles.wizardToggleListItem, status === 'error' && styles.modifiers.danger)}>
+            {status === 'error' ? (
+              <span className={css(styles.wizardNavLinkStatusIcon)}>
+                <ExclamationCircleIcon />
+              </span>
```
Comment (mcoker) [inline packages/react-core/src/components/Wizard/WizardToggle.tsx:101]: Doesn't look like the toggle is reflecting the error status

Screenshot 2024-05-17 at 7 02 26 PM

**Review by mcoker — APPROVED** [review]
@kmcfaul left some feedback. I think it's fine as-is if we want to merge it and follow up.

Re: the charts colors, I didn't review them all but I believe that's to be expected. We can review in depth with design but that can happen after this merges IMO.

File: packages/react-core/src/components/NotificationDrawer/__tests__/__snapshots__/NotificationDrawerHeader.test.tsx.snap [hunk 2]
```diff
@@ -6,7 +6,7 @@ exports[`NotificationDrawerHeader drawer header with count applied 1`] = `
     class="pf-v6-c-notification-drawer__header"
   >
     <h1
-      class="pf-v6-c-notification-drawer__header-title"
+      class="pf-v6-c-content--h1 pf-v6-c-notification-drawer__header-title"
```
[the remark this answers is not in the document]
Comment (kmcfaul) [reply packages/react-core/src/components/NotificationDrawer/__tests__/__snapshots__/NotificationDrawerHeader.test.tsx.snap]: Removed the Text wrapper for a plain h1

File: packages/react-core/src/components/Wizard/WizardToggle.tsx [hunk 3]
```diff
@@ -90,8 +92,14 @@ export const WizardToggle = ({
         aria-expanded={isNavExpanded}
       >
         <span className={css(styles.wizardToggleList)}>
-          <span className={css(styles.wizardToggleListItem)}>
-            <span className={css(styles.wizardToggleNum)}>{wizardToggleIndex}</span>{' '}
+          <span className={css(styles.wizardToggleListItem, isActiveStepStatus === 'error' && styles.modifiers.danger)}>
+            {isActiveStepStatus === 'error' ? (
+              <span className={css(styles.wizardNavLinkStatusIcon)}>
```
Comment (mcoker) [inline packages/react-core/src/components/Wizard/WizardToggle.tsx:97]: There is a toggle specific element for the status icon.

```suggestion
              <span className={css(styles. wizardToggleStatusIcon)}>
```

You can see the difference in alignment between a regular toggle item and the status item in this PR

Screenshot 2024-05-20 at 3 55 06 PM

Screenshot 2024-05-20 at 3 55 20 PM

What it should look like after updating the class on this element

Screenshot 2024-05-20 at 4 00 16 PM

File: packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx [hunk 4]
```diff
@@ -75,11 +85,19 @@ const cardData: CardData = {
       title: '10 Hosts',
       content: [
         {
-          icon: <ExclamationCircleIcon color={global_success_color_100.var} />,
+          icon: (
+            <Icon status="success">
+              <ExclamationCircleIcon />
+            </Icon>
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx:91]: I noticed we have a color/icon mismatch or two, so I [compared to core](https://pf6.patternfly.org/components/card/html-demos#aggregate-status-card) and am wondering if we can make some changes to match core.

```suggestion
            <Icon status="success">
              <CheckCircleIcon />
            </Icon>
```

File: packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx [hunk 5]
```diff
@@ -89,11 +107,19 @@ const cardData: CardData = {
...
+          ),
           count: 5
         },
         {
-          icon: <TimesCircleIcon color={global_danger_color_100.var} />,
+          icon: (
+            <Icon status="success">
+              <TimesCircleIcon />
+            </Icon>
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx:121]:
```suggestion
            <Icon status="danger">
              <ExclamationCircleIcon />
            </Icon>
```

File: packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx [hunk 6]
```diff
@@ -103,11 +129,19 @@ const cardData: CardData = {
...
+              <ExclamationTriangleIcon />
+            </Icon>
+          ),
           count: 3
         },
         {
-          icon: <TimesCircleIcon color={global_danger_color_100.var} />,
+          icon: (
+            <Icon status="danger">
+              <TimesCircleIcon />
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx:142]:
```suggestion
              <ExclamationCircleIcon />
```

File: packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx [hunk 7]
```diff
@@ -119,12 +153,20 @@ const cardData: CardData = {
       title: '13 Hosts',
       content: [
         {
-          icon: <TimesCircleIcon color={global_danger_color_100.var} />,
+          icon: (
+            <Icon status="danger">
+              <TimesCircleIcon />
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx:158]:
```suggestion
              <ExclamationCircleIcon />
```

File: packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx [hunk 8]
```diff
@@ -151,12 +201,20 @@ const cardData: CardData = {
...
+            </Icon>
+          ),
           status: '7 warnings',
           subtitle: 'subtitle'
         },
         {
-          icon: <TimesCircleIcon color={global_danger_color_100.var} />,
+          icon: (
+            <Icon status="danger">
+              <TimesCircleIcon />
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx:215]:
```suggestion
              <ExclamationCircleIcon />
```

File: packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx [hunk 9]
```diff
@@ -64,7 +70,11 @@ const cardData: CardData = {
       title: '3 Clusters',
       content: [
         {
-          icon: <TimesCircleIcon color={global_danger_color_100.var} />
+          icon: (
+            <Icon status="danger">
+              <TimesCircleIcon />
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardAggregateStatus.tsx:75]:
```suggestion
              <ExclamationCircleIcon />
```

File: packages/react-core/src/demos/examples/Card/CardStatus.tsx [hunk 10]
```diff
@@ -140,7 +140,7 @@ export const CardStatus: React.FunctionComponent = () => {
         <GridItem>
           <Flex spaceItems={{ default: 'spaceItemsSm' }}>
             <FlexItem>
-              <CheckCircleIcon color={global_success_color_100.var} />
+              <CheckCircleIcon color={global_color_status_success_default.var} />
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/Card/CardStatus.tsx:143]: any reason we wouldn't use `<Icon/>` for the icons in this file, too?

**Review by mcoker — APPROVED** [review]
Just one small change needed for the wizard toggle. 

Also left some comments on card demo icons but those are all pre-existing and just nits, so no importance to get in with this PR IMO.

Also there is a line in the react-tokens readme we could probably clean up - https://github.com/patternfly/patternfly-react/blob/90b90a0101dcd7417a4f0b4effde96af5b356f91/packages/react-tokens/README.md?plain=1#L25

^ references an old global var name and the filename in the import has hyphens that should be underscores. We could probably just copy/paste the `import` line from the block below it there:

```
import global_background_color_primary_default from '@patternfly/react-tokens/dist/esm/global_background_color_primary_default';
```

File: packages/react-core/src/components/Toolbar/ToolbarItem.tsx [hunk 11]
```diff
@@ -80,7 +80,8 @@ export const ToolbarItem: React.FunctionComponent<ToolbarItemProps> = ({
         <div
           className={css(
             styles.toolbarItem,
-            variant && styles.modifiers[toCamel(variant) as 'pagination' | 'label' | 'chipGroup'],
+            variant && styles.modifiers[toCamel(variant) as 'pagination' | 'label'],
+            variant === ToolbarItemVariant['chip-group'] && styles.modifiers.labelGroup,
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Toolbar/ToolbarItem.tsx:84]: Could we instead update the `chip-group` variant to `label-group` for the ToolbarItemVariant enum and prop type?

File: packages/react-core/src/demos/examples/Card/CardStatus.tsx (hunk 10 above)
Comment (kmcfaul) [reply packages/react-core/src/demos/examples/Card/CardStatus.tsx:143]: Nope I don't think so, I will update them here too. Any time I think we don't have to directly use tokens we probably should.

**Review by thatblindgeye — APPROVED** [review]
Above comment isn't a blocker and can be tackled in a followup

File: packages/react-core/src/components/Toolbar/ToolbarItem.tsx (hunk 11 above)
Comment (kmcfaul) [reply packages/react-core/src/components/Toolbar/ToolbarItem.tsx:84]: Opened https://github.com/patternfly/patternfly-react/issues/10405 to track that change in a follow up, along with the rest of Toolbar (as there are several usages of "chip" in its language for prop/component names).

**Comment by tlabaj** [comment]
> @tlabaj Noticed that we had one last usage of deprecated SelectOption in a deprecated Table editable cell demo. I updated it to use the new SelectOption, but it looks like that demo doesn't open the select menu. Do we want to open a follow up for this? Or are we planning on eventually removing deprecated Table in v6?

@kmcfaul Lets open a follow up for this for now.

**Review by tlabaj — APPROVED** [review]
lgtm

**Pull request merged**